### PR TITLE
Add todo

### PR DESCRIPTION
**Pull request by JasonEtco**
Add [todo](https://github.com/JasonEtco/todo), a Probot app that makes new issues from code comments. 🎉

### CODE REVIEW

File: _apps/todo.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: todo
+description: Creates new issues from actionable comments in your code.
+slug: todo
+screenshots:
+- https://user-images.githubusercontent.com/10660468/31048765-83569c30-a5f2-11e7-933a-a119d43ad029.png
+authors: [JasonEtco]
+repository: JasonEtco/todo
+stars: 10
+updated: 2017-09-30 05:57:19 UTC
```
Comment (hiimbex) [inline _apps/todo.md]: `stars` and `updated` are automatically synced so no need to list them here.

File: _apps/todo.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+`todo` would create a new issue:
+
+![todo](https://user-images.githubusercontent.com/10660468/31048765-83569c30-a5f2-11e7-933a-a119d43ad029.png)
+
+## Configuring for your project
+
+There are a couple of configuration options in case you need to change the default behaviour. Note that the defaults are likely fine for most projects, so you might not need to change them.
+
+Add a file called `todo.yml` in the `.github` folder at the root of your project.
```
Comment (hiimbex) [inline _apps/todo.md]: cc/ @bkeepers for thoughts on this one, but moving forward for our configuration patterns, I think I'd suggest using a `.github/config.yml` file that looks like this:
```yml
todo:
  autoAssign: true
  keyword: "@todo"
```

File: _apps/todo.md (hunk 1 above)
Comment (JasonEtco) [reply _apps/todo.md]: When do they get synced? At deploy time? I had built it locally and it didn't show anything :(

File: _apps/todo.md (hunk 2 above)
Comment (JasonEtco) [reply _apps/todo.md]: I'm +1 for a standardized way of doing things. If that's how we decide to go forward, we should either mention this in the docs or not give apps a choice on what the config file will be called (through `context.config()`)

**Comment by JasonEtco** [comment]
I've changed `todo` to use `.github/config.yml`: https://github.com/JasonEtco/todo/pull/5

**Comment by JasonEtco** [comment]
Alrighty I think this is good to merge @hiimbex if you want to officially approve it 💚 🍏 📗

**Comment by JasonEtco** [comment]
Ooh I'm not sure I should merge this, `octicons` might be a problem. cc @bkeepers for confirmation/merging 🙇

**Comment by hiimbex** [comment]
@JasonEtco that was fixed, so you can safely merge yourself! 😊 🚢

**Pull request merged**